src/coreclr/jit/lowerxarch.cpp

```diff
@@ -1985,19 +1942,24 @@ void Lowering::LowerHWIntrinsicCreate(GenTreeHWIntrinsic* node)
         GenTree* lo = comp->gtNewSimdHWIntrinsicNode(TYP_SIMD16, node->GetOperandArray(), halfArgCnt,
                                                      NI_Vector128_Create, simdBaseJitType, 16);
         BlockRange().InsertAfter(node->Op(halfArgCnt), lo);
-        LowerNode(lo);
+        lo = LowerNode(lo)->gtPrev;
 
         GenTree* hi = comp->gtNewSimdHWIntrinsicNode(TYP_SIMD16, node->GetOperandArray(halfArgCnt), halfArgCnt,
                                                      NI_Vector128_Create, simdBaseJitType, 16);
         BlockRange().InsertAfter(node->Op(argCnt), hi);
-        LowerNode(hi);
+        hi = LowerNode(hi)->gtPrev;
```

Comment:
Ditto: bashing and assumptions about `LowerNode` returns.

Reply:
Fixed.